app/(main)/league/[leagueId]/entry/[entryId]/week/Week.tsx

```diff
@@ -217,9 +221,19 @@ const Week = ({ entry, league, NFLTeams, week }: IWeekProps): JSX.Element => {
     };
 
     try {
-      await onWeeklyPickChange(params);
-      setUserPick(teamSelect);
-      router.push(`/league/${league}/entry/all`);
+      if (lockedOut) {
```

Comment:
I don't see any new unit tests for this.

Reply:
I will add these in ASAP.

Reply:
@shashilo can you elaborate what test exactly you're looking for?

Reply:
It's a conditional statement, so you need to have a positive and negative unit tests for this. Checking that both conditions are accurate when their conditions are met.